lib/internal/repl/await.js

```diff
@@ -33,8 +33,8 @@ const visitorsWithoutAncestors = {
     if (isTopLevelDeclaration(state)) {
       state.prepend(node, `${node.id.name}=`);
       ArrayPrototypePush(
-        state.hoistedDeclarationStatements,
-        `let ${node.id.name}; `
+        state.hoistedDeclaration,
+        `this${node.id.name} = ${node.id.name}; `
```

Comment:
Are you sure this didn't alter any other tests?

Reply:
Fixed,